### PR TITLE
Upgraded demo to use TF 0.12

### PR DESCRIPTION
**Pull request by wripley**
Tested using TF 0.12.3

### CODE REVIEW

File: terraform/provider.tf [hunk 1]
```diff
@@ -14,9 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-
 // Identifies allowable version range for Terraform Google Provider
 provider "google" {
-  project = "${var.project}"
-  version = "~> 1.13"
+  project = var.project
+  version = "~> 2.5"
```
Comment (chrislovecnm) [inline terraform/provider.tf]: 2.10 is the latest

**Review by chrislovecnm — COMMENTED** [review]
Just need to update the provider version to the latest, otherwise looks awesome

**Comment by wripley** [comment]
Ok to test.

**Review by chrislovecnm — DISMISSED** [review]
/lgtm

**Comment by chrislovecnm** [comment]
CI is not passing

**Review by bgeesaman — APPROVED** [review]
/lgtm
/approve

**Pull request merged**